src/hooks/useSearchSubmit.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from 'react'
```

Comment:
This hook is intended to be used in multiple places:
- Homepage search
- Header search